pygmt/datasets/map_tiles.py

```diff
@@ -0,0 +1,109 @@
+"""
+Function to load raster basemap tiles from XYZ tile providers, and load as
+:class:`xarray.DataArray`.
+"""
+
+try:
+    import contextily
+except ImportError:
+    contextily = None
+
+import numpy as np
+import xarray as xr
+
+__doctest_requires__ = {("load_map_tiles"): ["contextily"]}
+
+
+def load_map_tiles(region, source=None, lonlat=True, **kwargs):
+    """
+    Load a georeferenced raster basemap from XYZ tile providers.
+
+    The tiles that compose the map are merged and georeferenced into an
+    :class:`xarray.DataArray` image with 3 bands (RGB). Note that the returned
+    image is in a Spherical Mercator (EPSG:3857) coordinate reference system.
+
+    Parameters
+    ----------
+    region : list
+        The bounding box of the map in the form of a list [*xmin*, *xmax*,
+        *ymin*, *ymax*]. These coordinates should be in longitude/latitude if
+        ``lonlat=True`` or Spherical Mercator (EPSG:3857) if ``lonlat=False``.
+
+    source : xyzservices.TileProvider or str
+        [Optional. Default: Stamen Terrain web tiles] The tile source: web tile
+        provider or path to local file. The web tile provider can be in the
+        form of a :class:`xyzservices.TileProvider` object or a URL. The
+        placeholders for the XYZ in the URL need to be {x}, {y}, {z},
+        respectively. For local file paths, the file is read with rasterio and
```

Comment:
Please link `rasterio` to its site.

Reply:
Done at 5400710f38658f39b266da71622a35a549a1c076